lib/Command/Room/Create.php

```diff
@@ -92,14 +93,14 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
 		$owner = $input->getOption('owner');
 		$moderators = $input->getOption('moderator');
 
-		$name = trim($name);
-		if (!$this->validateRoomName($name)) {
+		$roomType = $public ? Room::PUBLIC_CALL : Room::GROUP_CALL;
+		try {
+			$room = $this->roomService->createConversation($roomType, $name);
+		} catch (\InvalidArgumentException $e) {
 			$output->writeln('<error>Invalid room name.</error>');
 			return 1;
 		}
```

Comment:
It is never really tested whether the room name is at fault... The fact that `InvalidArgumentException` is thrown for invalid names only right now is just a implementation detail of `RoomService::createConversation()`.

Reply:
Yeah, future "to be written" tests for the RoomService will make sure it throws on invalid room names

Reply:
Full tests written now

Reply:
Excellent :+1: 

However, my comment rather was about assuming that a `InvalidArgumentException` is thrown for invalid room names only. In other words: There's a `$e->getMessage() === 'name'` check missing.

Reply:
done